dev/com.ibm.ws.ejbcontainer.mdb/resources/com/ibm/ws/ejbcontainer/mdb/resources/EJBMDBMessages.nlsprops

```diff
@@ -66,3 +66,11 @@ MDB_ACTIVATION_SPEC_NOT_FOUND_CNTR4015W.useraction=Verify that an activation spe
 MDB_DESTINATION_NOT_FOUND_CNTR4016W=CNTR4016W: The message endpoint for the {0} message-driven bean cannot be activated because the {1} destination is not available. The message endpoint will not receive messages until the destination becomes available.
 MDB_DESTINATION_NOT_FOUND_CNTR4016W.explanation=The server is starting the message-driven bean but is unable to activate the message endpoint because the destination could not be found. The message endpoint will not receive messages until the endpoint can be activated.
 MDB_DESTINATION_NOT_FOUND_CNTR4016W.useraction=Verify that a destination has been configured with the specified name. If the configuration exists, wait for the destination to become available.
+
+MDB_ENDPOINT_ALREADY_INACTIVE_CNTR4117I=CNTR4117I: The message endpoint for the {0} message-driven bean was not paused because it was already in an inactive state.
+MDB_ENDPOINT_ALREADY_INACTIVE_CNTR4117I.explanation=The message endpoint can only be paused when it is in an active state.
+MDB_ENDPOINT_ALREADY_INACTIVE_CNTR4117I.useraction=No action is required.
+
+MDB_ENDPOINT_ALREADY_ACTIVE_CNTR4118I=CNTR4118I: The message endpoint for the {0} message-driven bean was not resumed because it was already in an active state.
```

Comment:
The ID reviewer will provide better comments here, but to me the use of past tense here seems a bit awkward,
... message-driven bean was not resumed because it was already in an active state.
One idea for rewording it:
... message-driven bean cannot be resumed because it is already active.
Alternatively, could rephrase the entire sentence as something like,
The resume operation did not have any effect because the message endpoint for the {0} message-driven bean is already active.
The same applies to the previous comment.

Reply:
I think the rephrasing you gave reads better, so I updated both messages.